CHANGELOG.md

```diff
@@ -1,8 +1,12 @@
 # Changelog
 
+## Unreleased
+
+- **Revert:** add `driving-traffic` profile to Isochrone service.
+
 ## 0.13.0
 
-- **Add:** add `driving-traffic` profile to Isochrone service.
+**Add:** add `driving-traffic` profile to Isochrone service.<br class="Apple-interchange-newline">
```

Comment:
```suggestion
**Add:** add `driving-traffic` profile to Isochrone service.
```

Reply:
@katydecorah Thank you so much for catching that!